### DISCUSSION TITLE
Button always available on page

### DISCUSSION THREAD

**Discussion by alexpoppe** [mcanouil/quarto-elevator#8] [Ideas]
Hello!

Thank you for the extension. Very easy to implement and clear instructions.

I was wondering if there is a possibility to keep the elevator button visible at all times on the screen.
This would allow the user to go back to the top even if he is only halfway through the page. 
The button would be hovering at the bottom of the screen wherever the user finds himself on the page and thus always be available.

I am quite new to quarto and don't know javascript well so please let me know if i am mistaken and this is actually already in place or just not all possible.

Thanks!

**Comment by mcanouil** [comment]
FYI, Quarto implements natively a back to top button for website project. (This was added after I created this extension).

https://quarto.org/docs/websites/website-navigation.html#back-to-top

I still plan to handle the native button but did not have the time just yet to do it.

- #5
